src/components/sidebar.tsx

```
@@ -0,0 +1,184 @@
+                          )}
+                        </div>
+                      </div>
+                    </div>
+                  </PopoverTrigger>
+                  <PopoverContent side="right">
+                    <div>
+                      <div className="flex space-x-3">
+                        <Avatar>
+                          <AvatarImage src={userData.avatar_url} />
+                          <AvatarFallback>
+                            {userData.name && userData.name.slice(0, 2)}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)